### DISCUSSION TITLE
Documentation Link to File is Broken

### DISCUSSION THREAD

**Discussion by erun264** [firefly-iii/firefly-iii#7723] [Firefly III] [answered]
Specifically the link to the docker-compose-data.yml file on (Alternative Docker Compose File)
 https://docs.firefly-iii.org/firefly-iii/installation/docker/ 

Gives a 404 Page not found error.

Is this an issue for others? Can the file be found elsewhere? I've not yet turned it up searching elsewhere.

**Comment by JC5** [comment, marked as answer]
https://raw.githubusercontent.com/firefly-iii/docker/main/docker-compose-importer.yml

The link is cached and should update soon.

**Reply by erun264** [reply]
Many thanks for the quick answer and help.
